core/commands/bitswap.go

```diff
@@ -244,29 +256,31 @@ prints the ledger associated with a given peer.
 	},
 }
 
-var reprovideCmd = &cmds.Command{
-	Helptext: cmds.HelpText{
+var reprovideCmd = &oldcmds.Command{
+	Helptext: cmdkit.HelpText{
 		Tagline: "Trigger reprovider.",
 		ShortDescription: `
 Trigger reprovider to announce our data to network.
 `,
 	},
-	Run: func(req cmds.Request, res cmds.Response) {
+	Run: func(req oldcmds.Request, res oldcmds.Response) {
 		nd, err := req.InvocContext().GetNode()
 		if err != nil {
-			res.SetError(err, cmds.ErrNormal)
+			res.SetError(err, cmdkit.ErrNormal)
 			return
 		}
 
 		if !nd.OnlineMode() {
-			res.SetError(errNotOnline, cmds.ErrClient)
+			res.SetError(errNotOnline, cmdkit.ErrClient)
 			return
 		}
 
 		err = nd.Reprovider.Trigger(req.Context())
 		if err != nil {
-			res.SetError(err, cmds.ErrNormal)
+			res.SetError(err, cmdkit.ErrNormal)
 			return
 		}
+
+		res.SetOutput(nil)
```

Comment:
Maybe we should have something in the code that calls `Run` check to see if `SetOutput` has not been called, and have it be smart in that case. Seems like a good safety fallback mechanism to avoid people forgetting to call this

Reply:
Though i guess this is *only* for the case of the old commands. in which case, i'm ambivalent.

Reply:
Yes, it's only for the old commands, and it's just those that don't output anything, which are just a handful.